licensing/install/protected/update_NEXT.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `SFXProvider` CHANGE `SFXProvider` `sfxProvider` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT; 
+ALTER TABLE `expression` CHANGE `documentText` `documentText` TEXT CHARACTER SET utf8 COLLATE utf8_general_ci NULL;
```

Comment:
The table name should be "Expression" (with capital "E").

Also note: In other upgrade scripts, we are using the command "ALTER TABLE [tablename] CONVERT TO CHARACTER SET utf8 COLLATE utf8_general_ci". We should confirm that these commands are equivalent.